oeis.py

```diff
@@ -169,6 +169,19 @@ def phi(n):
     return [phi(x) for x in range(start, start + limit)]
 
 
+@oeis
+def A023811(start,limit):
```

Comment:
Missing: Function docstring (see other functions). Test "oeis.py --list" to see why it's usefull.